### PR TITLE
fix(markdown-magic): Add missing "EXAMPLE" scope support, and correctly differentiate unrecognized scopes from lack of scope

### PR DESCRIPTION
**Pull request by Josmithr**
Also updates test assets to use recognized scope.

### CODE REVIEW

File: tools/markdown-magic/src/utilities.cjs [hunk 1]
```diff
@@ -121,7 +127,7 @@ const getScopeKindFromPackage = (packageName) => {
  */
 const shouldLinkToApiDocs = (packageName) => {
 	const scope = getScopeKindFromPackage(packageName);
-	return scope === "FRAMEWORK" || scope === "EXPERIMENTAL" || scope === undefined;
+	return scope === "FRAMEWORK" || scope === "EXPERIMENTAL" || scope === "";
```
Comment (Josmithr) [inline tools/markdown-magic/src/utilities.cjs:130]:
```suggestion
	return scope === "FRAMEWORK" || scope === "EXPERIMENTAL" || packageName === "fluid-framework";
```

File: tools/markdown-magic/src/utilities.cjs [hunk 2]
```diff
@@ -95,13 +95,19 @@ function getPackageMetadata(packageJsonFilePath) {
...
+ * @returns {"" | "FRAMEWORK" | "EXAMPLE" | "EXPERIMENTAL" | "INTERNAL" | "PRIVATE" | "TOOLS" | undefined}
+ * A scope kind based on the package's scope (namespace).
+ * Will be an empty string if the package has no scope.
+ * Will be `undefined` if the package has an unrecognized scope.
  */
 const getScopeKindFromPackage = (packageName) => {
 	const packageScope = PackageName.getScope(packageName);
-	if (packageScope === `@fluidframework`) {
+	if (packageScope === "") {
+		return undefined;
```
Comment (Josmithr) [inline tools/markdown-magic/src/utilities.cjs:106]:
```suggestion
		return "";
```

File: tools/markdown-magic/src/utilities.cjs [hunk 3]
```diff
@@ -95,13 +95,19 @@ function getPackageMetadata(packageJsonFilePath) {
...
+ * A scope kind based on the package's scope (namespace).
+ * Will be an empty string if the package has no scope.
+ * Will be `undefined` if the package has an unrecognized scope.
  */
 const getScopeKindFromPackage = (packageName) => {
 	const packageScope = PackageName.getScope(packageName);
-	if (packageScope === `@fluidframework`) {
+	if (packageScope === "") {
+		return "";
+	} else if (packageScope === `@fluidframework`) {
```
Comment (TommyBrosman) [inline tools/markdown-magic/src/utilities.cjs:107]: Why are these string literals written as template literals?

Comment (Josmithr) [reply tools/markdown-magic/src/utilities.cjs:107]: Good question. No idea. Certainly no reason for them to be so now. Will update.

**Review by alexvy86 — APPROVED** [review]

**Review by TommyBrosman — APPROVED** [review]

**Pull request merged**